data/members.yml

```diff
@@ -71,6 +71,11 @@ types:
          title: Codeplay Software
          link: https://www.codeplay.com/
          img_width: 150
+       -
+         image: /images/sponsors/csem-logo-cmyk.png
+         title: CSEM
+         link: https://www.csem.ch/Home
```

Comment:
I wanna confirm the site url as well

Reply:
CESM should be higher in alphabetic order :)

Reply:
@autumnfound Isn't this CSEM? 😅

Reply:
The url is confirmed.